demos/showcase/src/app/app.component.html

```diff
@@ -1,7 +1,96 @@
+<div style="display: flex; padding: 16px; justify-content: space-between">
+    <pxb-scorecard
+        [headerTitle]="'Substation 3'"
+        [headerSubtitle]="'High Humidity Alarm'"
+        [headerInfo]="'4 Devices'"
+        [headerColor]="colors.red[500]"
+        [headerFontColor]="colors.white[50]"
+    >
+        <mat-icon action-items>more_vert</mat-icon>
+        <mat-list body>
```

Comment:
Can you kill the extra top padding on all of these lists...particularly the actionRow ones since they mess up the vertical alignment